Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -69,25 +69,18 @@ open class PassiveLocationDataSource: NSObject {
             return
         }
         
-        directions.fetchAvailableOfflineVersions { [weak self] (versions, error) in
-            guard let self = self, let latestVersion = versions?.first(where: { !$0.isEmpty }), error == nil else {
-                completionHandler?(error)
-                return
-            }
-            
-            do {
-                try self.configureNavigator(withTilesVersion: latestVersion)
-                completionHandler?(nil)
-            } catch {
-                completionHandler?(error)
-            }
+        do {
+            try self.configureNavigator(withTilesVersion: nil)
+            completionHandler?(nil)
+        } catch {
+            completionHandler?(error)
```

Comment:
If not for backwards compatibility, this would be a good opportunity to make the surrounding method synchronous and throwable. We can make that change in v2.0.0.

/cc @chezzdev @Udumft